tests/cli/add_database_test.py

```diff
@@ -0,0 +1,79 @@
+
+
+from pathlib import Path
+import tempfile
+
+import pytest
+import yaml
+
+from datajunction.cli import add_database
+from datajunction.errors import DJException
+
+@pytest.mark.asyncio
+async def test_add_database_minimum_requirements() -> None:
+    """
+    Test adding a database with the minimum required arguments
+    """
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        await add_database.run(
+            Path(tmpdirname),
+            database="testdb",
+            uri="testdb://test",
+        )
+        with open(Path(tmpdirname) / Path("databases/testdb.yaml"), "r") as f:
+            assert yaml.safe_load(f) == {
+                "URI": "testdb://test",
+                "async_": False,
+                "cost": 1.0,  # default
+                "description": "",  # default
+                "read-only": True  # default
+            }
+
+@pytest.mark.asyncio
+async def test_add_database() -> None:
+    """
+    Test adding a database using required and optional arguments
+    """
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        await add_database.run(
+            Path(tmpdirname),
+            database="testdb",
+            uri="testdb://test",
+            description="This is a description",
+            read_only=True,
+            cost=11.0
```

Comment:
Python allow trailing commas and they're great, since they allow quickly reordering lines without having to move the comma around.

```suggestion
            cost=11.0,
```

We use an autoformatter that should take care of this.